mobile/src/main/java/rs/readahead/washington/mobile/javarosa/FormUtils.java

```diff
@@ -26,6 +27,18 @@
 
 
 public class FormUtils {
+    public static boolean isTellaLocationHiddenField(Context context, FormEntryPrompt fep) {
+        String locationFieldPrefix = context.getString(R.string.tella_location_field_prefix);
+        String fepName = fep.getIndex().getReference().getNameLast();
+        return (fepName.startsWith(locationFieldPrefix));
+    }
+
+    public static boolean isTellaMetadataHiddenField(Context context, FormEntryPrompt fep) {
```

Comment:
Maybe DRY this a little as both `isTellaLocationHiddenField` and `isTellaMetadataHiddenField` use same code with different prefixes. Also, there are parenthesis leftovers on return clauses.